### DISCUSSION TITLE
Github - understanding when there are new posts

### DISCUSSION THREAD

**Discussion by KryptoNyte-Sage** [OpenSageTV/Community#71] [General]
I've noticed that there can be significant new activity in any given thread, but there doesn't seem to be any real indication that it has occurred.  What is the best way to see all the "new activity" here in the SageTV Github forum?  Am I missing a button somewhere that I can click on and see all new activity in a glance?

**Comment by eelpout** [comment]
as far as I can tell, there is no way to filter just new posts. so, from the main discussion page, I look at the sorting type and then use the filter ("Sort by" or "Top...") to look at the "Top Past Day" or past week.

**Reply by eelpout** [reply]
going to reply to myself. yeah, this doesn't really work the way I expected it to. went back to just poking around to find new stuff. :/

**Comment by tiki-2** [comment]
I agree - the GitHub forums are terrible for this.  With the old forums there was a view that showed all unread posts sorted by date.  When you clicked on one, it took you to the first unread post in that thread.  It was very easy to stay on top of things and be sure that you saw every post.

with GitHub, I have been using a view that sorts by latest activity with filter set to All.  But it doesn’t seem to really work right.  It seems to show threads that have unread posts in a darker / bolder font, but they can pop up anywhere in the list (I would expect them to be at the top of the list, but they aren’t).  When you click one of these, it takes you to the first post and doesn’t seem to have any indicator to show which posts you’ve already read, so you are forced to re-read all the posts to try to find what’s new (and sometimes there doesn’t appear to be anything new at all).  Posts aren’t always in order within the thread - there seems to be a concept of replies and comments that I don’t fully comprehend.

While I used to check the old forums multiple times per day, I’m finding this new forum so frustrating that I rarely check-in anymore.  Based on the volume of posts that I see, I’m guessing I’m not the only one.

**Reply by Ugadata** [reply]
I agree - this is a Reply to a Comment

This shows up indented, directly under the Comment.

**Comment by Ugadata** [comment]
I agree - This is a Comment.

Comments show up at the bottom of the Thread(?)

**Comment by wnjj** [comment]
The replies show up under a specific post if you use the “write a reply” box.  They get added to the bottom if you use the “write” box below all the posts.

I find the format pretty terrible too.  Sometimes the new activity is buried deep in a chain of replies on a post near the top and other times it’s at the bottom.

The desktop site “recent activity” highlighting seems different on the web version than the mobile app too.

**Comment by KryptoNyte-Sage** [comment]
I just saw another new thing today that I hadn't seen before.  When replies to a specific post, that occurs right below the post itself as others have mentioned, reach a certain quantity of replies, that it starts to hide the most recent replies.  So when you enter a thread that apparently has recent activity, you can't just find that most recent activity by scrolling, sometimes you have to expand the truncated replies to further complicate things.

What a strange format.

**Comment by LoisGNS** [comment]
My approach has been to subscribe to all new posts. That means I get all sorts of posts that aren't relevant to me, but given the pending potential program with Schedules Direct, I don't want to miss anything.

**Comment by KryptoNyte-Sage** [comment]
Is there a way to quote another post, or is the absence of quoting why the reply system works the way it does?

**Comment by eelpout** [comment]
> Is there a way to quote another post, or is the absence of quoting why the reply system works the way it does?

click on the "three dots" in the upper right and there is a "Quote reply" option.

Edit: But oddly, it's not a "reply" to your direct post. I really don't understand this thing at all. :)

**Reply by KryptoNyte-Sage** [reply]
This is a quoted reply, testing.

**Reply by KryptoNyte-Sage** [reply]
Still not getting a quote.

**Comment by KryptoNyte-Sage** [comment]
> > Is there a way to quote another post, or is the absence of quoting why the reply system works the way it does?
> 
> click on the "three dots" in the upper right and there is a "Quote reply" option.
> 
> Edit: But oddly, it's not a "reply" to your direct post. I really don't understand this thing at all. :)

Oh, here we go.
